.travis.yml

```diff
@@ -16,13 +16,14 @@ os:
   - linux
   - osx
 osx_image: xcode7.1
+before_install:
+  - if test "$TRAVIS_OS_NAME" == "osx"; then brew update; brew install nginx; fi
 install:
-  - curl -sSL https://github.com/libuv/libuv/archive/v1.4.2.tar.gz | tar zxfv - -C /tmp && cd /tmp/libuv-1.4.2/
-  - sh autogen.sh
-  - ./configure --prefix=$HOME/libuvinstall
+  - curl -sSL http://nginx.org/download/nginx-1.8.0.tar.gz | tar zxfv - -C /tmp && cd /tmp/nginx-1.8.0/
```

Comment:
Is this also going to run on OSX? If so, why are we `brew install`ing nginx and then building it from source?